frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/RebaseOnParentBackgroundable.java

```diff
@@ -86,73 +92,77 @@ private GitRebaseParams getIdeaRebaseParamsOf(GitRepository repository, IGitReba
   @Override
   @UIThreadUnsafe
   public void run(ProgressIndicator indicator) {
```

Comment:
Speaking of rebase in traverse...

push dialog should be scheduled on UI thread, not executed immediately, so that the actions can update in the meantime

Reply:
I do not understand how it could work - the traverse is side effecting and e.g. rebase too. how will traverse "allow" to rebase?

Reply:
Nvm... just drunken mumblings, ignore